src/functions.cpp

```diff
@@ -1,9 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include <algorithm>
 #include <iostream>
 #include <limits>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (long unsigned int i = 0; i < numbers.size(); ++i) {
```

Comment:
You can consider using `size_t` instead of `long unsigned`

Reply:
Indeed. Thanks  ;)